### PR TITLE
Only display the workflow status from the most recent version

### PR DESCRIPTION
**Pull request by jcoyne**
Also:
* Extract a presenter for the workflow processes

### CODE REVIEW

File: app/presenters/workflow_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+class WorkflowPresenter
+  def initialize(object:, workflow_name:, xml:)
+    @object = object
+    @workflow_name = workflow_name
+    @xml = xml
+  end
+
+  delegate :pid, to: :object
+  delegate :processes, to: :workflow_document
```
Comment (mjgiarlo) [inline app/presenters/workflow_presenter.rb]: It looks like you're both delegating and defining `#processes`. Won't one make the other dead code?

Comment (jcoyne) [reply app/presenters/workflow_presenter.rb]: Ah, good call. I thought I removed this. I suspect it's just noop, because the other one overwrites this.

**Comment by mjgiarlo** [comment]
@jcoyne 	restarted the failing build. Feel free to merge when all :green_heart:

File: app/presenters/workflow_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  def pretty_xml
+    # rubocop:disable Rails/OutputSafety
+    CodeRay::Duo[:xml, :div].highlight(Nokogiri::XML(xml).prettify).html_safe
+    # rubocop:enable Rails/OutputSafety
+  end
+
+  private
+
+  attr_reader :object, :xml, :workflow_steps
```
Comment (mjgiarlo) [inline app/presenters/workflow_presenter.rb]: Are the private attrs for `:object` and the `:workflow_steps` used, or are they cruft?

Comment (jcoyne) [reply app/presenters/workflow_presenter.rb]: used on lines 15 & 24 respectively

Comment (mjgiarlo) [reply app/presenters/workflow_presenter.rb]: :eyes: 	:roll_eyes:  :zzz:

**Pull request merged**